src/ts/tinymce.ts

```diff
@@ -104,14 +105,14 @@ function doneTyping(): void {
 
 // options for tinymce to pass to tinymce.init()
 export function getTinymceBaseConfig(page: string): object {
-  let plugins = 'accordion advlist anchor autolink autoresize table searchreplace code fullscreen insertdatetime charmap lists save image link pagebreak codesample template mention visualblocks visualchars emoticons';
+  let plugins = 'accordion advlist anchor autolink autoresize table searchreplace code fullscreen insertdatetime charmap lists save image media link pagebreak codesample template mention visualblocks visualchars emoticons';
   let toolbar1 = 'undo redo | styles fontsize bold italic underline strikethrough | alignleft aligncenter alignright alignjustify | superscript subscript | bullist numlist outdent indent | forecolor backcolor | charmap emoticons adddate | codesample | link | sort-table | save';
-  let removedMenuItems = 'newdocument, image, anchor';
+  let removedMenuItems = 'newdocument, image, media, anchor';
```

Comment:
is there a reason to remove it from the menu items?

Reply:
To be honest I just followed the way it's done for the image plugin because the media plugin is doing basically the same thing just for video/audio links. So no, no specific reason beside that.

Reply:
then I suggest leaving it, so it is accessible from the menu bar and the menu: so remove it from `removedMenuItems`

Reply:
FYI: now that this is all merged, I noticed that the icon is gone (see images) with this change and media is only accessible via the menu

Before:
![Screenshot_20240807_132851](https://github.com/user-attachments/assets/187cc2dd-05bd-4015-98d2-f1f28645ce97)
After:
![Screenshot_20240807_132551](https://github.com/user-attachments/assets/ec7e5d95-bab2-4bce-9e15-627c05004626)

So perhaps that was the reason why this has been done to the image plugin in the first place?

Reply:
Nevermind... that's an oversight on my end. Didn't add it to the toolbar after removing from the conditional.

Reply:
Yes, I thought it was on purpose. And the toolbar being already plentiful of icon I feel it's best if this functionality stays only available from the menu.